config/initializers/session_store.rb

```diff
@@ -1,4 +1,6 @@
 # frozen_string_literal: true
 # Be sure to restart your server when you modify this file.
 
-Rails.application.config.session_store :cookie_store, key: "_velum_session"
+Rails.application.config.session_store :cookie_store,
+  key:    "_velum_session",
+  secure: Rails.env.production?
```

Comment:
We can also set it on development, but not on the test environment as tests are going to fail, maybe we can change it to `secure: !Rails.env.test?` ?